helm/helm.go

```diff
@@ -81,6 +81,10 @@ func main() {
 					Name:  "force, aye-aye",
 					Usage: "Perform install even if dependencies are unsatisfied.",
 				},
+				cli.BoolFlag{
+					Name:  "dry-run",
+					Usage: "Run but do not install into Kubernetes.",
```

Comment:
I think what @bacongobbler was getting at in help is that this will do the `fetch` part if necessary, but will not run the `kubectl` part. We should probably make that clear.

I'm wondering if it makes sense to pass the dryRun flag into `uploadManifests` and have it print out what it would normally do. That would give the user a chance to see whether the final generated JSON is what they anticipate.

Thoughts, @sgoings ?